### PR TITLE
Adding create Pull Request Template #75

### PR DESCRIPTION
**Pull request by lalitmee**
**Contributing Screenshot**
pull1

**Pull Request Template Screenshot**
pull2

### CODE REVIEW

**Review by gauthamzz — COMMENTED** [review]
I don't think you need to update readme . It's fine . Only change the pull request template

**Comment by gauthamzz** [comment]
You can add a contributing.md for that part. Add it in docs

**Comment by gauthamzz** [comment]
In pull request template . Add follow commit me rules . See the one by github .

**Comment by lalitmee** [comment]
Okk. Can you provide the link to rules by GitHub.

**Comment by lalitmee** [comment]
Are you talking about [this](https://github.com/erlang/otp/wiki/writing-good-commit-messages)?

**Comment by gauthamzz** [comment]
leave it ,its fine the way you send.
you can add one line, add relevant screenshots or images if needed

**Comment by lalitmee** [comment]
Tell me I can do it. Now I have created a **contributing.md** file in docs.

File: docs/PullRequestTemplate.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Pull Request Template #
+
+Please follow these practices while contributing to Anna.
+
```
Comment (gauthamzz) [inline docs/PullRequestTemplate.md:4]: write here

- Make sure you add screenshots if applicable

File: docs/README.md [hunk 2]
```diff
@@ -31,10 +30,29 @@ Watch Anna in action on [Youtube](https://www.youtube.com/watch?v=17bVrAZMgEY):
...
 
+- Avoid duplicate PRs, if need be comment on the older PR with the PR number of the follow-up (new PR) and close the obsolete PR yourself.
+
+- Add relevant screenshots or images if needed.
+
+Also, we have a team of maintainers to whom you can write if your pull-request goes unnoticed. They may have time.
+
+[https://anna-assistant.github.io/](https://anna-assistant.github.io/)
+
+You can talk to us on  [Zulip](https://anna.zulipchat.com/) or [Tweet](https://twitter.com/anna_assistant) or comment on issues to contribute.
```
Comment (gauthamzz) [inline docs/README.md]: Dont change the readme . add these contents in contributing.md

**Comment by lalitmee** [comment]
screenshot from 2017-12-30 22-13-20

File: docs/PullRequestTemplate.md (hunk 1 above)
Comment (gauthamzz) [reply docs/PullRequestTemplate.md:4]: This looks good now.
Dont update the readme. And its fine

**Comment by lalitmee** [comment]
Now, is it okk?

**Review by gauthamzz — APPROVED** [review]

File: docs/README.md (hunk 2 above)
Comment (lalitmee) [reply docs/README.md]: You want these two lines in contributing.md?

Comment (gauthamzz) [reply docs/README.md]: no its perfect now, nothing needs to be done

Comment (lalitmee) [reply docs/README.md]: Okk.

**Pull request merged**